scheduledao.py

```diff
@@ -0,0 +1,8 @@
+from dao import DAO
+
+class ScheduleDAO(DAO):
```

Comment:
As above, just ensure that the DAO class is abstract enough to not affect the outcome of either of these classes .. if we make a change I the DAO it should be ok to be applied to Schedule and Event